pages/money.tsx

```diff
@@ -0,0 +1,20 @@
+import { useUser } from '@auth0/nextjs-auth0'
+import React from 'react'
+
+import BaseLayout from '../components/BaseLayout'
+import FinanceDateHeader from '../components/FinanceDateHeader'
+
+export default function Money() {
+  const { user, isLoading, error } = useUser()
+
+  if (isLoading) return <div>Loading...</div>
+  if (error) return <div>{error.message}</div>
+
+  // TODO: add useOrientation like custom hook.
+  const left = true ? -5 : 0
```

Comment:
what are these numbers? Not able to understand from the code

Reply:
this is for responsiveness, which can be done separately